oauthproxy.go

```diff
@@ -814,7 +814,7 @@ func (p *OAuthProxy) OAuthCallback(rw http.ResponseWriter, req *http.Request) {
 		}
 		http.Redirect(rw, req, redirect, http.StatusFound)
 	} else {
-		logger.PrintAuthf(session.Email, req, logger.AuthFailure, "Invalid authentication via OAuth2: unauthorized")
+		logger.PrintAuthf(session.Email, req, logger.AuthFailure, "Invalid authentication via OAuth2: unauthorized login")
```

Comment:
Why diverge from the upstream ` oauth2-proxy/oauth2-proxy` with this change?

In the future we're going to want to pull changes, do we really have to rebase this change forever, maybe resolve conflicts when they change this line too?